Automattic/woocommerce-payments
# Allow only Japanese phone numbers for Japanese accounts

Fixes #6911

#### Changes proposed in this Pull Request
An error message is displayed when a non-Japanese phone is used for Japanese accounts in the `Support phone` field on the `Settings` page.

<img width="340" alt="Screenshot 2023-08-03 at 14 50 07" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/89e0437f-76e4-4ce2-9379-236f3bb4cc96">

<img width="521" alt="Screenshot 2023-08-03 at 14 50 28" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/308333b5-4518-4fb0-8dba-34a36d5b5104">

#### Testing instructions
- Use a Japanese account.
- Navigate to `Payments` -> `Settings`.
- Verify only Japan is available on the `Support phone` input field.
- Enter a non-Japanese phone, e.g. '+359878111111'.
- Verify an error is triggered on Save settings.

-------------------


**Post merge**
